crates/revm/src/handler/mainnet.rs

```diff
@@ -107,6 +107,7 @@ pub fn calculate_gas_refund<SPEC: Spec>(env: &Env, gas: &Gas) -> u64 {
 }
 
 #[cfg(test)]
+#[cfg(not(feature = "optimism"))]
```

Comment:
This is not needed, 
```suggestion
#[cfg(not(feature = "no_gas_measuring"))]
```

Reply:
Left remove `USE_GAS` from `handle_call_return` it would not disrupt anything and would help with tests. `no_gas_measuring` has been shown to be just an annoyance without much benefit so it is probable that it will be removed.